AnagramSolver.BusinessLogic/Classes/WordRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AnagramSolver.Contracts.Interfaces;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace AnagramSolver.BusinessLogic.Classes
+{
+    public class WordRepository : IWordRepository
+    {
+        public HashSet<string> GetAllWords()
+        {
+            HashSet<string> newList = new HashSet<string>();
+            HashSet<string> allLines = new HashSet<string>(File.ReadLines(@"C:\Users\jonas.jaugelis\source\repos\AnagramSolver\zodynas.txt"));
```

Comment:
Avoid using hardcoded values / paths. This path perfectly fits in `appsettings.json` file.

Reply:
done